tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
+    "WinObjC.Frameworks": "*",
+    "WinObjC.Frameworks.UWP": "*",
+    "WinObjC.Frameworks.Core": "*",
+    "WinObjC.Language": "*",
+    "WinObjC.Logging": "*"
```

Comment:
There is a dependency chain described already. The only difference is what version is picked up due to the constraints being satisfied with an older version or not. We should expect the typical case to just depend on the tip even if our internal samples etc depend on all to force the latest of everything even in prerelease mode.